apps/gql/clients/__mocks__/clients.ts

```diff
@@ -1,6 +1,5 @@
-import { type Clients } from "../types";
 import { mockedPrisma } from "./prisma";
 
-export const mockedClients: Clients = {
-  prisma: mockedPrisma
-};
\ No newline at end of file
+export const mockedClients = {
```

Comment:
what about typing this still. I see the type has changed below so we can still use the type.

Reply:
removing this actually makes typing work. if we type this as `Clients` test files will not know `findMany` has a function called `mockReturnValueOnce`